beetmoverscript/task.py

```diff
@@ -87,9 +88,16 @@ def generate_checksums_manifest(context):
     return '\n'.join(content)
 
 
+def is_custom_beetmover_task(context):
```

Comment:
Do we plan on using this function outside of checksums? If not, adding `checksums` somewhere in the name may help clarify its purpose.

Reply:
Good point, I've addressed it now in a follow-up commit: https://github.com/mozilla-releng/beetmoverscript/pull/130/commits/c7d542e8ae5d44db0597b0b8a603c01059112262